app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -149,6 +151,112 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
+  @Test
+  fun testExploration_overflowMenu_isDisplayedSuccessfully() {
+    getApplicationDependencies(TEST_EXPLORATION_ID_2)
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+    ).use {
+      openActionBarOverflowOrOptionsMenu(context)
+      onView(withText(context.getString(R.string.menu_options))).check(matches(isDisplayed()))
+      onView(withText(context.getString(R.string.help))).check(matches(isDisplayed()))
+    }
+    explorationDataController.stopPlayingExploration()
+  }
+
+  @Test
+  fun testExploration_openOverflowMenu_selectHelpInOverflowMenu_showsHelpFragmentSuccessfully() {
+    getApplicationDependencies(TEST_EXPLORATION_ID_2)
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+    ).use {
+      openActionBarOverflowOrOptionsMenu(context)
+      onView(withText(context.getString(R.string.help))).perform(click())
+      onView(
```

Comment:
Instead of this, you should check if the intent information is correct or not:

Check this test case for reference: `testHomeActivity_recyclerViewIndex1_clickPromotedStory_opensTopicActivity`

Reply:
Done